how-to/workspace-native-window-integration/client/src/platform-storage.ts

```diff
@@ -1,43 +1,43 @@
 export class PlatformStorage {
```

Comment:
I don't think we need this file as it was updated in the last merge.

Reply:
This is in the NWI example not customize-workspace